openedx/openedx-platform
# feat: export tagged content library as csv [FC-0049]

## Description
This PR implements a feature to export a CSV with the library structure and its blocks, showing all tags applied to them.


## More information
Part of:
- https://github.com/openedx/modular-learning/issues/173

Depends on:
- https://github.com/openedx/edx-platform/pull/34091

## Testing instructions
1. Check the tests cover the implemented feature
1. Follow the instructions from https://github.com/openedx/frontend-app-library-authoring/pull/412 to test it from the MFE

___
Private-ref: [FAL-3611](https://tasks.opencraft.com/browse/FAL-3611)
